### PR TITLE
Merge dev to main

### PR DESCRIPTION
**Pull request by mmahdavian**
merging dev branch to main for the first time.

### CODE REVIEW

File: scenario_runner/srunner/metrics/data/DistanceToLaneCenter_criteria.json
Comment (pagand) [inline scenario_runner/srunner/metrics/data/DistanceToLaneCenter_criteria.json:1]: should not be deleted!

File: scenario_runner/srunner/metrics/data/DistanceBetweenVehicles_criteria.json
Comment (pagand) [inline scenario_runner/srunner/metrics/data/DistanceBetweenVehicles_criteria.json:1]: should not be deleted!

File: scenario_runner/srunner/metrics/data/CriteriaFilter_criteria.json
Comment (pagand) [inline scenario_runner/srunner/metrics/data/CriteriaFilter_criteria.json:1]: should not be deleted!

File: scenario_runner/srunner/data/routes_training.xml
Comment (pagand) [inline scenario_runner/srunner/data/routes_training.xml:1]: should not be deleted!

File: scenario_runner/srunner/data/routes_devtest.xml
Comment (pagand) [inline scenario_runner/srunner/data/routes_devtest.xml:1]: should not be deleted!

File: scenario_runner/srunner/data/routes_debug.xml
Comment (pagand) [inline scenario_runner/srunner/data/routes_debug.xml:1]: should not be deleted!

File: scenario_runner/srunner/data/no_scenarios.json
Comment (pagand) [inline scenario_runner/srunner/data/no_scenarios.json:1]: should not be deleted!

File: scenario_runner/srunner/data/all_towns_traffic_scenarios1_3_4_8.json
Comment (pagand) [inline scenario_runner/srunner/data/all_towns_traffic_scenarios1_3_4_8.json:1]: should not be deleted!

File: scenario_runner/srunner/data/all_towns_traffic_scenarios1_3_4.json
Comment (pagand) [inline scenario_runner/srunner/data/all_towns_traffic_scenarios1_3_4.json:1]: should not be deleted!

File: scenario_runner/srunner/data/all_towns_traffic_scenarios.json
Comment (pagand) [inline scenario_runner/srunner/data/all_towns_traffic_scenarios.json:1]: should not be deleted!

File: leaderboard/team_code/x13/model_main.py
Comment (pagand) [inline leaderboard/team_code/x13/model_main.py:1]: What is this?

Comment (mmahdavian) [reply leaderboard/team_code/x13/model_main.py:1]: just remove it

**Review by pagand — COMMENTED** [review]
undeate the files

**Comment by mmahdavian** [comment]
done

**Review by pagand — APPROVED** [review]

**Pull request merged**